packages/toggle/src/types.ts

```diff
@@ -0,0 +1,56 @@
+import { Either, HTMLElementProps } from '@leafygreen-ui/lib';
+
+export const Size = {
+  Default: 'default',
+  Small: 'small',
+  XSmall: 'xsmall',
+} as const;
+
+export type Size = typeof Size[keyof typeof Size];
+
+interface BaseToggleProps {
+  /**
+   * Sets the size of the toggle.
+   *
+   * default: `'default'`
```

Comment:
small tsdoc nit: this should be `@default 'default'` (i.e. `@` symbol, no colon, and no extra quotes)